get-involved/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+hero-image: url(../assets/images/stock-images/pie-staff.jpg)
+hero-message: Get Involved!
+---
+
+<!-- WHAT WE DO -->
+<div class="container">
+    <h1 style="text-align: center">Join PiE Staff</h1>
+    <!--<h2>WHAT WE DO</h2>-->
+</div>
+<div class="rc-row vertical-align" >
+    <div class="text-col text-left" id="fall-row" >
+        <div class="graphic-description">
+            <h3>ENGINEERING</h3>
+            <p>We're mechanical engineers, software engineers, electrical engineers. We build, design, reiterate, and
+            improve with industry-grade products.</p>
+            <h2><i class="fa fa-wrench"></i> <i class="fa fa-laptop"></i> <i class="fa fa-cogs"></i></h2>
+        </div>
+    </div>
+    <div class="text-col text-right" id="spring-row" >
+        <div class="graphic-description">
+            <h3>OUTREACH</h3>
+            <p>We're UC Berkeley students who care about making a difference in the lives of underrepresented students in science and engineering.</p>
+            <h2><i class="fa fa-heart"></i> <i class="fa fa-users"></i> <i class="fa fa-book"></i></h2>
+        </div>
+    </div>
+</div>
+<div id="fall-row" class="rc-row vertical-align">
+    <div id="rc-video" class ="image-col embed-responsive embed-responsive-16by9">
```

Comment:
The Engineering/Outreach section above this is split 50/50, but here, the video and the text next to it is not 50/50, and it looks a little off?